README.md

```diff
@@ -48,6 +37,20 @@ We are still happy to review new contributions to `docker-maven-plugin`, but we
 do not plan on making major new contributions ourselves in the future.
```

Comment:
Perhaps change the wording here as well.

Reply:
I'll just remove this paragraph since it's redundant.